payload is not part of the header of a message

Arttaaz opened the pull request.

Arttaaz committed:
payload is not part of the header of a message

ghost commented:
Je suis d’accord pour enlever le champ payload qui n’a effectivement rien à faire là, mais il faut dans ce cas là préciser d’une façon ou d’une autre que celle-ci viens à la suite du header à un endroit.

Arttaaz committed:
Update messages.md

Modification de la présentation des messages pour aussi inclure la présentation de la payload

Arttaaz commented:
Précision faite.

ghost commented:
Pour moi c’est bon, il nous faudrait l’avis d’un autre contributeur. @traxys @ValentinDebon @jonathanjulou @AurelienMoisson

traxys commented:
+1
